instrumentation/active_support/lib/opentelemetry/instrumentation/active_support/span_subscriber.rb

```diff
@@ -125,6 +128,16 @@ def sanitized_value(value)
             value
           end
         end
+
+        # Helper method to try an shield the span name formatter from errors
+        #
+        # It wraps the user supplied formatter in a rescue block and returns the original name if a StandardError is raised by the formatter
+        def safe_span_name_for(span_name_formatter, name)
+          span_name_formatter&.call(name) || name
```

Comment:
If we're going to go this route, should we `#dup#freeze` the `name` that we fall back to for consistency with how the legacy formatter now works?

Reply:
I added `dup.freeze` to the result of the method call.